### ISSUE TITLE
Improve README

### ISSUE THREAD

**Issue by lucsedirae** [lucsedirae/open-house-crm#52]
Current README version is a placeholder. As the styling develops and our final "look" comes together for version 1, include screenshots, gifs, expand on usage, include info on libraries/APIs/etc that are employed.

**Comment by suchitapallavi** [comment]
@lucsedirae Hey, I am new to open source. I have learned Html, CSS and bootstrap. Can you please guide me that what exact changes you want in your README? Also, there is no installation instruction. So can you please tell me what installation process should I follow?

**Comment by lucsedirae** [comment]
Hello @suchitapallavi ! Thanks for reaching out. This project is actually our final group project in a boot camp we are taking. We won't be accepting contributions until after we present and finish the boot camp but I would love to have some help from that point forward. I can reach out once we've gotten to that point (about 3 weeks from now) and we can connect on where we'll need help from there.

**Comment by lucsedirae** [comment]
The plan is to continue maintaining and building this out post boot camp, so really excited to have some interest already from other devs. We're all new to web dev and I think it would be great to build this tool out with a lot of "new" developers contributing and learning.

**Comment by suchitapallavi** [comment]
Hey, let me know whenever you are going to start it. I am very excited to
contribute.

Regards
Suchita Pallavi
https://suchitapallavi.com

**Comment by lucsedirae** [comment]
Will do @suchitapallavi . We have about 10 days left and will be opening it up for outside contributions. Feel free to clone the repo down and play around with it. You will need to create a mongoDB Atlas account and add a mongoURI to a default.json file that should be stored in the config folder. 

Follow the Atlas instruction when you log in to create a cluster and collection and then get a personalized URI and create a "secret" that can also be stored in the default.json file. This will allow you to store data remotely in your own Mongo database and have access to manage it through their web portal. The default.json file should look like this:
```
{
    "mongoURI": "mongodb+srv://<yourMongoDBUserName>:<yourPassword>@cluster0.dy---1.mongodb.net/collection_name?retryWrites=true&w=majority", 
    "jwtSecret": "<secure randomized string>"
}
```

**Comment by lucsedirae** [comment]
@suchitapallavi Also, fyi, I'm closing out this issue but feel free to message me through GitHub or via email if you have questions or want to stay in touch. jondeavers@gmail.com

**Comment by lucsedirae** [comment]
@suchitapallavi Hi there. We've finally wrapped up our presentations and have had a little time to catch our breath. If you're still interested in helping out with the project, I'd recommend joining a discord server I set up as a place for our class to hang out. Now that boot camp is over we've opened it up to any web developer who wants to network, learn, or help others. Send me an email or a twitter DM and I'll send over a server invite and that will be a good place to communicate on where we could use some help. jondeavers@gmail.com or @jondeavers on Twitter.
